<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Scientist Database</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Create Person ( Debug )</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Create Computer ( Debug )</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="middle_group">
      <item>
       <widget class="QGroupBox" name="people_group">
        <property name="title">
         <string>People</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="filter_group_p">
           <item>
            <layout class="QVBoxLayout" name="filter_name_p_g">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Filter by name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filter_name_p">
               <property name="placeholderText">
                <string>Filter name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_gender_p_g">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Filter by gender:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="filter_gender_p">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_birth_p_g">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Filter by birth:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="filter_birth_c_p">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QDateEdit" name="date_birth_p"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_death_p_g">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Filter by death:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="filter_death_c_p">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QDateEdit" name="date_death_p"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="people_list">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear_person">
           <property name="text">
            <string>Clear Selection</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="people_button_group">
           <item>
            <widget class="QPushButton" name="pushButton_delete_person">
             <property name="text">
              <string>Delete Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_modify_person">
             <property name="text">
              <string>Modify Selected</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="computer_group">
        <property name="title">
         <string>Computers</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="filter_group_c">
           <item>
            <layout class="QVBoxLayout" name="filter_group_name_c_g">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Filter by name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filter_name_c">
               <property name="placeholderText">
                <string>Filter name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_group_type_c_g">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Filter by type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filter_type_c">
               <property name="placeholderText">
                <string>Filter type</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_group_year_c_g">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Filter by year:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="filter_date_c_c">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QDateEdit" name="date_date_c">
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="filter_group_built_c_g">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Filter by built:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="filter_built_c"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="computer_list">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear_computer">
           <property name="text">
            <string>Clear Selection</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="computer_button_group">
           <item>
            <widget class="QPushButton" name="pushButton_delete_computer">
             <property name="text">
              <string>Delete Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_modify_computer">
             <property name="text">
              <string>Modify Selected</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionCreate_Person"/>
    <addaction name="actionCreate_Computer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuEntry"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_Person">
   <property name="text">
    <string>Person</string>
   </property>
  </action>
  <action name="actionCreate_Computer">
   <property name="text">
    <string>Computer</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Delete_Person">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_Modify_Person">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="action_Delete_Computer">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_Modify_Computer">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
